nearcore/src/state_sync.rs

```diff
@@ -209,19 +209,31 @@ fn get_current_state(
     account_id: &Option<AccountId>,
     epoch_manager: Arc<dyn EpochManagerAdapter>,
 ) -> Result<Option<(EpochId, EpochHeight, CryptoHash)>, Error> {
```

Comment:
nit: I can't tell from the return type what's what. Can you change it to a dedicated enum with meaningful names for the available options?